lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -15,6 +13,7 @@ Gem::Specification.new do |spec|
 <%- if config[:mit] -%>
   spec.license       = "MIT"
 <%- end -%>
+  spec.required_ruby_version = Gem::Requirement.new(">= 1.9")
```

Comment:
IMHO if we're going to do this it should be 2.1, which is the oldest MRI version still receiving updates.